src/components/DataDictionary.tsx

```diff
@@ -90,6 +90,7 @@ export const columns: ColumnDescriptor[] = [
     name: "category",
     label: "Category",
     rowValueDescriptor: "category",
+    // eslint-disable-next-line react/display-name
```

Comment:
Bleh!

Looks like eslint doesn't like these formatter functions. Another reason to refactor this functionality